server/cluster/cluster_worker.go

```diff
@@ -77,13 +77,17 @@ func (c *RaftCluster) HandleAskSplit(request *pdpb.AskSplitRequest) (*pdpb.AskSp
 	return split, nil
 }
 
-// ValidRequestRegion is used to decide if the region is valid.
-func (c *RaftCluster) ValidRequestRegion(reqRegion *metapb.Region) error {
+// ValidRequestSplitRegion is used to decide if the region is valid.
+func (c *RaftCluster) ValidRequestSplitRegion(reqRegion *metapb.Region) error {
```

Comment:
Does the region size become larger if it is a hot write region?

Reply:
If such a bug occurs, it needs to be resolved. Currently, this limitation will be judged in both PD and TiKV.